isofit/radiative_transfer/modtran.py

```diff
@@ -268,8 +269,30 @@ def readSim(self, point):
         For a given point, parses the tp6 and chn file and returns the data
         """
         file = os.path.join(self.sim_path, self.point_to_filename(point))
+        try:
+            solzen = self.load_tp6(f"{file}.tp6")
+        except FileNotFoundError as e:
+            # Rerun the point until it is done
+            done = False
+            n_rerun = 0
+            while not done:
+                # Pause for a second
+                time.sleep(np.random.randint(1, 10) / 10.0)
+                # Extract filename that doesn't exist
+                logging.info(f"File not found: {e.filename}")
+                if n_rerun >= 10:
+                    logging.info(f"{n_rerun} reruns; stopping")
+                    raise FileNotFoundError(e)
+                    # Throw error
+                self.makeSim(point)  # Rerun
+                # Try to load
+                try:
+                    solzen = self.load_tp6(f"{file}.tp6")
+                    done = True
+                except FileNotFoundError:
+                    pass
+                n_rerun += 1
```

Comment:
I don't like this as a general case - in the limit (badly parameterized MODTRAN case), you would have to run each LUT simulation 10 times prior to failure, which is way expensive for a failure state.  @reginaeckert (I believe this was your addition) - do you know what's causing these failures?

Reply:
Let's try and add a check w/ the license server status as an easy way around.

Reply:
For now, I removed this try/except clause entirely. We could think of adding a check with the license server status at a later time.